pennylane/fourier/visualize.py

```diff
@@ -313,7 +313,9 @@ def circuit_with_weights(w, x):
         axis.boxplot(
             data[data_type],
             boxprops=dict(
-                facecolor=to_rgb(data_colour) + (0.4,), color=data_colour, edgecolor=data_colour
+                facecolor=to_rgb(data_colour) + (0.4,),
```

Comment:
The changes in this file are all just from `black` right?

Reply:
Yes, sorry! I'm wondering why this is? Maybe my `black` is outdated?

Reply:
Resolved.